roles/docker/tasks/debian.yml

```diff
@@ -1,26 +1,34 @@
 ---
+- name: Ensure old versions of Docker are not installed.
+  package:
+    name: '{{ item }}'
+    state: absent
+  with_items:
+    - docker
+    - docker-engine
 
 - name: install apt-transport-https
   apt:
     name: apt-transport-https
     state: installed
```

Comment:
Should this line be `state: present` or just deleting it as `present` is the default choice? According to Ansible official document, I don't think `installed` is the valid choice http://docs.ansible.com/ansible/latest/apt_module.html